Cargo.toml

```diff
@@ -20,3 +23,4 @@ rand = "0.5"
 [features]
 unsafe_internals = []
 reverse_bits = []
```

Comment:
I don't think we need `reverse_bits` and `no_reverse_bits`. Personally I'd delete both, and just silently use reverse bits; but if we really, really want an option I think `no_reverse_bits` is enough.

Reply:
deleted the features.